slackapi/deno-slack-sdk
# Fix circular import in builtin generation script

### Summary

In the builtin generation script there seems to be a circular import where `src/schema/slack/functions/_scripts/src/write_function_files.ts` indirectly imports `src/schema/slack/functions/mod.ts`. The issue steams from `src/schema/slack/functions/_scripts/generate` since it is cleaning all the files from `src/schema/slack/functions` before executing `src/schema/slack/functions/_scripts/src/write_function_files.ts` which lead to the following error.

```txt
error: Module not found "file:///~/deno-slack-sdk/src/schema/slack/functions/mod.ts".
    at file:///~/deno-slack-sdk/src/schema/slack/mod.ts:2:28
``` 

This PR introduces a change to `_scripts/generate` that temporarily creates a dummy `functions/mod.ts` file before executing `_scripts/src/write_function_files.ts`

### testing

1. Ensure you are using Deno 1.36.0 or higher (`deno -V`)
2. Pull the `test-builtin-generation` branch
3. Follow the instructions in `src/schema/slack/functions/_scripts/README.md` to generate the latest builtin
4. The generation should work properly

### Special notes

I also updated the comment generation function to point users to the correct README.md file path

### Requirements 

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/deno-slack-sdk/blob/main/.github/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've ran `deno task test` after making the changes.
